web-ui/src/components/reviews/TeamMemberReview-old.jsx

```diff
@@ -0,0 +1,367 @@
+import React, { useContext, useState, useCallback } from 'react';
```

Comment:
Micheal wants to keep the previous version of this code for possible use in the future.